adyen.php

```diff
@@ -1136,6 +1144,7 @@ private function getStandardPaymentMethod()
     private function isUnsupportedPaymentMethod($paymentMethodType)
     {
         $unsupportedPaymentMethods = array(
+            'scheme',
```

Comment:
Is this on purpose?

Reply:
yes - but wrong branch
I will create another branch to fix some issues - I just noticed that the card payments are included anyway and doesn't check the paymentMethods response so we show the scheme 2 times if it is in the paymentMethods list which was never noticed because the isSimplePaymentMethod function filtered the original one. :(